doc-content/jbpm-docs/src/main/asciidoc/CoreEngine/deployment-versions-con.adoc

```diff
@@ -0,0 +1,72 @@
+[id='deployment-versions-con_{context}']
+= Deployments and different process versions
+
+The deployment service puts business assets into an execution environment. However, in some cases additional management is required to make the assets available in the right context. Notably, if you deploy several versions of the same process, you must ensure that process instances use the correct version.
+
+[float]
+== Activation and Deactivation of deployments
+
+In some cases, a number of process instances are running on a deployment, and then you add a new version of the same process to the runtime environment.
+
+You might decide that new instances of this process definition must use the new version while the existing active instances should continue with the previous version.
+
+To enable this scenario, use the following methods of the deployment service:
+
+* `activate`: activates a given deployment so it can be available for interaction. You can lost its process definitions and start new process instances for this deployment.
+
+* `deactivate`: deactivates a deployment. Disables the option to list process definitions and to start new process instances of processes in the deployment. However, you can continue working with the process instances that are already active, for example, signal events and interact with user tasks.
+
+You can use this feature for smooth transition between project versions without the need for process instance migration.
+
+[float]
+== Invoking latest version of  processes
```

Comment:
Suggest "Invocation of the latest version of a process" (non-proc, and for consistency)